src/txmempool.h

```diff
@@ -260,7 +260,7 @@ struct TxMempoolInfo
  *
  * Usually when a new transaction is added to the mempool, it has no in-mempool
  * children (because any such children would be an orphan).  So in
- * addUnchecked(), we:
+ * addNewTransaction(), we:
  * - update a new entry's setMemPoolParents to include all in-mempool parents
```

Comment:
not your fault but: I don't think `setMemPoolParents` exists in master even

I feel like these paragraphs are a minefield of old information

Reply:
Tried to remove all instances of setMemPoolParents and setMemPoolChildren from code comments.